src/main/webapp/csrf.jsp

```diff
@@ -1,43 +1,9 @@
-<%@ page language="java" contentType="text/html; charset=UTF-8"
-	pageEncoding="UTF-8"%>
-<!DOCTYPE html>
-<html>
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
-<link rel="stylesheet" type="text/css"
-	href="css/bootstrap-theme.min.css">
-<link rel="stylesheet" type="text/css" href="css/style.css">
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
 
-<title>Spiracle - CSRF</title>
-</head>
-
-<body>
-	<div class="navbar navbar-default navbar-fixed-top" role="navigation">
-		<div class="container">
-			<div class="navbar-header">
-				<button type="button" class="navbar-toggle collapsed"
-					data-toggle="collapse" data-target=".navbar-collapse">
-					<span class="sr-only">Toggle navigation</span> <span
-						class="icon-bar"></span> <span class="icon-bar"></span> <span
-						class="icon-bar"></span>
-				</button>
-				<a class="navbar-brand" href="index.jsp">Spiracle</a>
-			</div>
-			<div class="navbar-collapse collapse">
-				<ul class="nav navbar-nav">
-					<li><a href="index.jsp">Overview</a></li>
-					<li><a href="file.jsp">File</a></li>
-					<li><a href="network.jsp">Network</a></li>
-					<li><a href="sql.jsp">SQL</a></li>
-                    <li><a href="xss.jsp">XSS</a></li>
-                    <li class="active"><a href="csrf.jsp">CSRF</a></li>
-                    <li><a href="pathTraversal.jsp">Path Traversal</a></li>
-                    <li><a href="misc.jsp">Misc</a></li>
-				</ul>
-			</div>
-		</div>
-	</div>
+  <jsp:include page="header.jsp" >
+    <jsp:param name="pageName" value="CSRF" />
+    <jsp:param name="fileName" value="csrf" />
```

Comment:
This param doesn't seem to be used anywhere in the change set. Is it required in any subsequent changes?